roles/openshift_management/defaults/main.yml

```diff
@@ -3,6 +3,8 @@
 openshift_management_project: openshift-management
 # Namespace/project description
 openshift_management_project_description: CloudForms Management Engine
+# Number of retries when waiting for the app to start (retried every 30 seconds)
+openshift_management_retries: 30
```

Comment:
Let's keep this at 20, what it is today, unless you think 30 should be the actual minimum.

Also, I don't like the name of this variable.

How about "openshift_management_pod_rollout_retries"

Reply:
The ManageIQ image is about 2.2GB.
To download it in less than 10 minutes the node it installs on would need a connection faster than 30Mb/s to the docker registry, and then ManageIQ itself takes a while to seed its database.

So yeah, I think 30 retries (15 minutes) should be the minimum.

I agree regarding the variable name and will change it.

Reply:
@elad661 ok, thanks for the information, that sounds good to me.

I would just like the variable name change then.

Reply:
I've already changed the name :)